CHANGELOG.md

```diff
@@ -0,0 +1,36 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
```

Comment:
Current changelog don't quite match this format :sweat_smile:

Reply:
It's close enough. We don't need all the fanciness that keepachangelog is suggesting IMO